cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/repo.go

```diff
@@ -114,13 +114,15 @@ func (s *Server) filterReadyReposByName(repoList []sourcev1.HelmRepository, matc
 
 	resultKeys := sets.String{}
 	for _, repo := range repoList {
+		repo := repo // avoid implicit memory aliasing
```

Comment:
One option to avoid memory aliasing is to use a shadowed variable, but seeing the instruction `repo := repo` looks strange to me at first sight (maybe coming from a Java world :).

In some places are using:
```
for i := range repoList {
    repo := repoList[i]
    ...
}
```
Just my observation, though.

Reply:
Certainly does look less odd. Overview of options here: https://stackoverflow.com/a/68247837